blocks/rich-text/index.js

```diff
@@ -558,6 +560,21 @@ export class RichText extends Component {
 		if ( keyCode === BACKSPACE ) {
 			this.onChange();
 		}
+
+		this.scrollToCaret();
+	}
+
+	scrollToCaret() {
+		const caretRect = this.getEditorSelectionRect();
+		const caretHeight = caretRect.y;
+		if ( caretHeight !== this.caretHeight ) {
```

Comment:
This compares to an instance value, meaning on its first invocation, it will always call `window.scrollTo`. Do we need the instance value, or can we just compare against the desired `pageYOffset` to see if we need to perform a scroll?

Reply:
Good point. Actually, one thing we can do is:

```js
if ( caretHeight !== this.caretHeight ) {
  const delta = caretTop - toolbarOffset;
  if ( Math.abs( delta ) > epsilon ) {
    window.scrollTo( window.pageXOffset, window.pageYOffset + delta );
  }
}
```

where _epsilon_ is as an arbitrarily small number, e.g. 10.

This solves the problem of pointless scrolling more generally.